index.html

```diff
@@ -0,0 +1,142 @@
+                  <svg
+                    width="15"
+                    height="20"
+                    viewBox="0 0 15 20"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M6.54751 20C11.2442 20 14.1667 16.5392 14.1667 12.0742C14.1667 10.3908 13.8925 8.87083 11.9817 8.41417C12.4467 7.14083 13.7883 3.4575 13.7883 2.2775C13.7883 1.23833 13.1633 0.3675 12.0492 0.3675C10.2733 0.3675 8.68167 5.51167 8.18334 6.73833C7.7975 5.625 5.99084 0 4.39834 0C3.16084 0 2.58001 0.894167 2.58001 2.01917C2.58001 3.39833 3.94084 6.9725 4.43417 8.3475C4.18417 8.2575 3.91334 8.17917 3.6375 8.17917C2.70584 8.17917 1.715 9.31583 1.715 10.2342C1.715 10.5817 1.91 11.07 2.03334 11.3942C0.565005 11.785 4.77463e-06 12.7458 4.77463e-06 14.195C-0.00416189 17.015 2.71917 20 6.54751 20ZM10.5725 4.21917C10.7325 3.73833 11.4125 1.7075 12.0492 1.7075C12.3917 1.7075 12.4833 2.055 12.4833 2.3325C12.4833 3.07833 10.9475 7.2 10.6083 8.11333L9.255 7.87917L10.5725 4.21917ZM3.83 1.8875V1.88667C3.83 1.42167 4.40751 0.101667 5.67251 3.72667L7.05 7.645C6.42917 7.59417 5.94751 7.5275 5.64167 7.7C5.20751 6.575 3.83 3.02417 3.83 1.8875ZM3.73834 9.53083L3.73917 9.53167C4.905 9.53167 6.41001 13.2267 6.41001 13.7267C6.41001 13.9258 6.215 14.1717 5.98834 14.1717C5.15584 14.1717 2.92667 11.1675 2.92667 10.355C2.93001 10.0542 3.43167 9.53083 3.73834 9.53083ZM2.39834 12.5892C2.85167 12.5892 4.55917 14.945 4.55917 15.445C4.55917 15.6367 4.2525 15.7692 4.09417 15.7692C3.45334 15.7692 3.2025 15.1633 2.06084 13.7617C0.878338 14.9217 2.87667 17.1558 4.38167 17.1558C5.42001 17.1558 6.09667 16.2108 5.89417 15.515C6.04167 15.515 6.22417 15.5267 6.36 15.4917C6.40417 16.55 6.7225 17.8117 8.02001 17.9017C8.02001 17.8667 8.10001 17.6242 8.10001 17.6125C8.10001 16.9325 7.67834 16.3392 7.67834 15.6475C7.67834 14.5417 8.54167 13.4717 9.41751 12.8467C9.73584 12.6125 10.1225 12.4675 10.4958 12.335C10.8825 12.1908 11.2925 12.0225 11.5858 11.7333C11.5417 11.2958 11.3592 10.9092 10.9133 10.9092C9.81084 10.9092 6.11334 11.0658 6.11334 9.35833C6.11334 9.09667 6.11751 8.84667 6.80667 8.84667C8.09251 8.84667 11.3567 9.15917 12.3117 9.98333C13.0317 10.6125 13.2792 14.405 11.0775 16.8075L11.075 16.8083C9.91667 18.0583 8.43584 18.7067 6.70834 18.7067C4.34417 18.7067 2.47667 17.4333 1.5775 15.2575C0.896672 13.5625 1.72917 12.5892 2.39834 12.5892ZM7.14917 11.8858C7.53667 12.0075 7.93417 12.0433 8.33251 12.1208C8.03751 12.3317 7.77584 12.59 7.52417 12.8667C7.41251 12.535 7.27667 12.2108 7.14917 11.8858Z"
+                      fill="#505F79"
+                    />
+                  </svg>
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="header-shapes">
+            <img src="images/header-shapes mobile.png" alt="header-shapes" />
+          </div>
+        </section>
+
+        <!-- hero section ends-->
+      </div>
+    </header>

```

Comment:
**[OPTIONAL]** - To help improve your site's accessibility and SEO benchmarks, the `header` tag should only wrap up your navigation. In your case, it can wrap up both the status bar and navigation. You mage find this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) useful in learning more about the `header` tag 😄

Reply:
Thank you @mckent05 - I will surely use the feedback to improve my skills.